evennia/contrib/base_systems/ingame_reports/README.md

```diff
@@ -0,0 +1,128 @@
+# In-Game Reporting System
+
```

Comment:
See the [layout of a contrib](https://www.evennia.com/docs/latest/Contribs/Contribs-Guidelines.html#layout-of-a-contrib) and add `Contrib by Inspector Caracal 2024`.